rootfs/Makefile

```diff
@@ -8,4 +8,5 @@ all:
 			sudo ./mkrootfs_debian.sh --arch "$${arch}" --distro "$${distro}" ; \
 		done \
 	done
+	sudo ./mkrootfs_ubuntu.sh --arch riscv64 --distro jammy
```

Comment:
why not just aligning on the distro? As ,uch as CI will be concerned, having different rootfs distro will be a nightmare. There is #84 which is a separate item to handle more rootfs distros.

Reply:
You mean align the other builds to use Ubuntu (Jammy or later)? RISC-V is still missing stable Debian support, so currently Ubuntu Jammy and later is the only option for RV. 

I agree that having different distros is not desirable.

Reply:
> You mean align the other builds to use Ubuntu (Jammy or later)? RISC-V is still missing stable Debian support, so currently Ubuntu Jammy and later (or Debian Sid) is the only option for RV.
> 
> I agree that having different distros is not desirable.

That would work once we move the runners to Jammy, which is not the case.
At the moment, we build for Debian Bullseye on Ubuntu Focal.... it happens to work, but this is by luck. Jammy target would not work. The problem is due to build dependencies changes between host and target OS. #83 explains the problem.